app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -12,6 +12,8 @@ module HostgroupExtensions
 
         validates_with Katello::Validators::ContentViewEnvironmentValidator
 
+        attr_accessible :content_view_id, :lifecycle_environment_id
```

Comment:
We have some hostgroup controller extensions that override the foreman create action but it still relies upon protected attributes (and not strong_params). Not sure if it's possible to strong_params in this case since Houstgroup is a Foreman model.